src/components/brew-input.js

```diff
@@ -16,18 +16,18 @@ const FirstPanel = (props) => {
       <FancyDropdown label="Brew Method " name="brewMethod" handleChange={props.handleChange}
         options={testEquip}
         />
-      <FancyInput label="Brew Date" name="brewDate" handleChange={props.handleChange}
-        value={props.brewDate} />
-      <FancyButton name="dateButton"
-        handleSubmit={props.setDateToday}
-        //handleSubmit={()=>console.log("date")}
-        buttonText="Set Date to Today"/>
+      <FancyDayPickerInput
+          label="Brew Date"
+          name="brewDate"
+          placeholder={props.brewDate}
+          onDayChange={props.handleDayChange}
+        />
+      <FancyButton name="dateButton" onClick={props.setDateToday} buttonText="Set Date to Today"/>
```

Comment:
Hi, long time no see. But I'm wondering where do I put in `value={props.value}`?

<img width="477" alt="screen shot 2017-07-26 at 5 44 45 pm" src="https://user-images.githubusercontent.com/12474808/28647198-3e612820-722b-11e7-94f6-21b44b8f7c54.png">

Not **in** `DayPickerInput` right? but outside of it?